products/redhat-jboss-eap.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Red Hat JBoss Enterprise Application Platform
+category: server-app
+iconSlug: redhat
+permalink: /redhat-jboss-eap
+alternate_urls:
+-   /redhat-jboss-eap
+-   /redhatjboss
+-   /redhatjbosseap
+
+versionCommand: $JBOSS_HOME/bin/version.sh
+releasePolicyLink: https://access.redhat.com/support/policy/updates/jboss_notes#phases
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+releaseDateColumn: true
+extendedSupportColumn: Extended Life Cycle Support
+
+releases:
+-   releaseCycle: "7"
+    support: 2023-06-30
+    eol: 2025-06-30
+    extendedSupport: 2029-11-30
+    latest: "7.4.6"
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
+    latestReleaseDate: 2023-03-01
+    link: https://access.redhat.com/articles/6716841
+
+-   releaseCycle: "6"
+    support: 2016-06-30
+    eol: 2019-06-30
+    extendedSupport: 2025-06-30
```

Comment:
Should be December 30, 2028 ?

Reply:
Working on it

Reply:
https://access.redhat.com/articles/3751821

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/91e7058f-cf76-42c4-a4e2-1a6168f6f991)

Reply:
According to this : 

- End of ELS-1 : 2022-06
- End of ELS-2 : 2025-06

Which one should we keep ?

Reply:
According to https://access.redhat.com/support/policy/updates/jboss_notes#phase_compare no security or bug fixes are issued in ELS 2, so I would say ELS-1. This is also something that must be documented in the product description.